edenia/antelope-tools
# Feat/next bps list update 1171

###  Next update of the BPs list

### What does this PR do?

- Resolve #1171 
- Add the `producers_list_update_log` table to store the last update and the next estimated update
- For the estimation, the date of the last update is taken and add the time to re-execute the list update plus the estimated time that the process will take, this being the sum of the maximum time to obtain the producers list from the blockchain and to get the bp.jsons.

### Steps to test

1. Run the project locally
2. Go to Block Producers, Nodes and Endpoints page
3. Check that the last update and next estimated update are displayed

### Screenshot

# Desktop

![imagen](https://user-images.githubusercontent.com/66583677/236513742-5f5fe099-4859-4296-8f33-19e0e88535b7.png)
![imagen](https://user-images.githubusercontent.com/66583677/236514245-bb3cadc5-3251-4a2b-aebd-be401eb93bcb.png)
![imagen](https://user-images.githubusercontent.com/66583677/236514340-964556d8-e875-4cd1-b410-77c2088f9c4d.png)

# Mobile

![imagen](https://user-images.githubusercontent.com/66583677/236514391-f75f7861-0a8f-4007-877e-885fca464c8e.png)
![imagen](https://user-images.githubusercontent.com/66583677/236514035-c0710ab5-8daf-4f67-9d53-e5ca41915447.png)
![imagen](https://user-images.githubusercontent.com/66583677/236514099-ef3b7968-6a09-4e91-a99c-7fc2d53f823c.png)
